src/Build/Graph/ProjectInterpretation.cs

```diff
@@ -227,6 +230,8 @@ ProjectItemInstance projectReference
             // The properties on the project reference supersede the ones from the MSBuild task instead of appending.
             if (newProperties.Count == 0)
             {
+                // This mimics the _GetProjectReferenceTargetFrameworkProperties task in order to properly reflect what the build graph looks like in
```

Comment:
```suggestion
                // This mimics the _GetProjectReferenceTargetFrameworkProperties target
```

It's not just for dynamic platform scenarios